san/sanbase_graphql.py

```diff
@@ -640,3 +667,8 @@ def _format_to_date(datetime_obj_or_str):
     dt = iso8601.parse_date(datetime_obj_or_str) + \
         datetime.timedelta(hours=23, minutes=59, seconds=59)
     return dt.isoformat()
+
+def _format_return_fields(fields):
+    return list(map(
+        lambda el: el[0] + '{{' + ' '.join(el[1]) + '}}' if isinstance(el, tuple) else el
```

Comment:
it's possible to have deeply nested fields - this won't handle them properly. what happens if you have a tuple that contains a tuple?

Reply:
You're right, that's a good question

Reply:
Should be okay now